modules/stochastic_tools/test/tests/actions/parameter_study_action/cartesian_product.i

```diff
@@ -0,0 +1,12 @@
+[ParameterStudy]
+  input = sub.i
```

Comment:
This syntax looks pretty nice. I think we can extend along this line for branching/homogenization-equivalence calculations.

Reply:
That's good to hear!